### ISSUE TITLE
The 'Run at Login' option is not enabled.

### ISSUE THREAD

**Issue by dotmanner** [AdguardTeam/AdguardForMac#1333]
### AdGuard version

2.12.2.1415

### Browser version

Firefox 119.0

### OS version

MacOS Sonoma 14.1

### What filters do you have enabled?

AdGuard Base filter, AdGuard Tracking Protection filter, AdGuard URL Tracking filter, AdGuard Social Media filter, AdGuard Annoyances filter, EasyList, EasyPrivacy, Peter Lowe's Blocklist, Online Malicious URL Blocklist, Fanboy's Enhanced Tracking List, List-KR, I don't care about cookies, EasyList Cookie List, Others

### What Stealth Mode options do you have enabled?

Hide your search queries, Send Do-Not-Track signals, Strip URLs of tracking parameters, Self-destruction of third-party cookies, Self-destruction of first-party cookies, Block Push API, Block Location API, Block Java, Remove X-client-Data header

### Support ticket ID

83162

### Issue Details

Steps to reproduce:
1. Enable the 'Run at Login' option in Settings and reboot.
2. ADGUARD is not running automatically.
3. Run ADGUARD manually.
4. Go into settings and the option is unchecked.

### Expected Behavior

After logging in and entering the desktop, ADGUARD should launch and an icon should appear in the top bar.

### Actual Behavior

Nothing happens - the option is unchecked.

### Screenshots

<details><summary>Screenshot 1:</summary>

</details>

**Comment by AlexandrPkhm** [comment]
Hi @dotmanner 

Please make sure that AdGuard and AdGuard Software Limited (if present) are allowed to run in the background. You can do this by opening System Settings → General → Login Items. 
![Screenshot 2023-11-13 at 12 16 44](https://github.com/AdguardTeam/AdguardForMac/assets/109953785/8a4607e6-3836-4f89-bc30-b3a183089aa1)

Also, let us know if you are using any applications that can manipulate login items on your device.

**Comment by dotmanner** [comment]
I solved the problem. In my case, running the **sfltool resetbtm** command in Terminal worked. There was a ghost entry in Settings > Allow in background that wouldn't be deleted related to AdGuard (it remained in the list even though I deleted all the related files from the relevant folders) and after running this command, the list was clean and after a clean install of AdGuard, everything worked fine.
